doc/mysql.sql

```diff
@@ -7,7 +7,7 @@ CREATE TABLE users (
     email     VARCHAR(511)   DEFAULT '',
     lastLogin TIMESTAMP      DEFAULT CURRENT_TIMESTAMP,
     lastNotif TIMESTAMP      DEFAULT CURRENT_TIMESTAMP,
-    opts      VARCHAR(16383) DEFAULT '{}'
+    opts      VARCHAR(15000) DEFAULT '{}'
```

Comment:
What's up here?

Reply:
Wouldn't allow more then a certain amount all together for the table, forgot the number

Reply:
'Error Code: 1118. Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs'